docs/source/debug_accuracy.md

```diff
@@ -0,0 +1,110 @@
+# Step by step example how to debug accuracy with Neural Insights
+1. [Introduction](#introduction)
+2. [Environment preparation](#environment-preparation)
+3. [Running the quantization](#running-the-quantization)
+4. [Analyzing the result of quantization](#-analyzing-the-result-of-quantization)
+5. [Analyzing weight histograms](#-analyzing-weight-histograms)
+
+# Introduction
+In this instruction accuracy issue will be debugged using Neural Insights. TensorFlow Inception_v3 model will be used as an example. It will be quantized and the results will be analyzed to find the cause of the accuracy loss.
+
+# Environment preparation
+## Install Intel® Neural Compressor and Neural Insights from source
+First you need to install Intel® Neural Compressor.
+```shell
+# Install Neural Compressor
+git clone https://github.com/intel/neural-compressor.git
+cd neural-compressor 
+pip install -r requirements.txt 
+python setup.py install
+
+# Install Neural Insights
+pip install -r neural_insights/requirements.txt
+python setup.py install neural_insights
+```
+
+## Install requirements for inception_v3
+```shell
+cd examples/tensorflow/object_detection/tensorflow_models/quantization/ptq
+pip install -r requirements.txt
+```
+
+## Prepare the model
+Download pre-trained PB model file.
+```shell
+wget https://storage.googleapis.com/intel-optimized-tensorflow/models/v1_6/inceptionv3_fp32_pretrained_model.pb
+```
+
+## Prepare the dataset
+Download dataset from ImageNet and process the data to TensorFlow Record format.
+```shell
+cd examples/tensorflow/image_recognition/tensorflow_models/
+bash prepare_dataset.sh --output_dir=./inception_v3/quantization/ptq/data --raw_dir=/PATH/TO/img_raw/val/ --subset=validation
+bash prepare_dataset.sh --output_dir=./inception_v3/quantization/ptq/data --raw_dir=/PATH/TO/img_raw/train/ --subset=train
+```
+
+# Running the quantization
+Before applying quantization, modify some code to enable Neural Insights:
+1. Set the argument `diagnosis` to be `True` in `PostTrainingQuantConfig` so that Neural Insights will dump weights and activations of quantizable Ops in this model.
+2. Denote the `op_name_dict` argument because that’s the answer of our investigation.
```

Comment:
`conf = PostTrainingQuantConfig(calibration_sampling_size=[50, 100], diagnosis=True)`
Add the code change.    

`Denote` --> `Delete`